apps/meteor/client/views/admin/cloud/RegisterWorkspace.tsx

```diff
@@ -43,7 +42,7 @@ const RegisterWorkspace = () => {
 		<Page background='tint'>
 			<Page.Header title={t('Registration')}>
 				<RegisterWorkspaceMenu
-					isWorkspaceRegistered={isWorkspaceRegistered}
+					isWorkspaceRegistered={isRegistered || false}
```

Comment:
I didn't get the usage of `false` here

Reply:
This page is going to be removed too.. I've just added to prevent ts error